ic-http/src/transform.rs

```diff
@@ -83,4 +84,48 @@ mod test {
         // Assert.
         assert_eq!(names, vec!["transform_function_1", "transform_function_2"]);
     }
+
+    /// Transform function which is intentionally creates a new request
+    /// with itself as a transform function.
```

Comment:
This sentence is a bit hard to parse. Maybe rewrite it a bit? Something like:
```
Transform function which intentionally creates a new request passing itself as the target transform function.
```

Reply:
Done in #185.